index.html

```diff
@@ -1,389 +1,386 @@
-
-      <!-- CONTACT SECTION -->
-      <section class="white-bg">
-        <div class="contact-section">
-          <h2 class="contact-heading">Contact me</h2>
-          <p class="contact-para">
-            If you have an application you are interested in developing, a
-            feature that you need built or a project that needs coding. I’d love
-            to help with it
-          </p>
-          <form
-            action="https://formspree.io/f/mzboowzg"
-            method="post"
-            class="contact-form"
-          >
-            <ul>
-              <li>
-                <label for="name"></label>
-                <input
-                  type="text"
-                  id="name"
-                  name="username"
-                  tabindex="4"
-                  maxlength="30"
-                  placeholder="Ghazanfar Ali"
-                  aria-label="name"
-                  required
-                />
-              </li>
-              <li>
-                <label for="email"></label>
-                <input
-                  type="email"
-                  id="email"
-                  name="useremail"
-                  tabindex="5"
-                  placeholder="gz123@gmail.com"
-                  aria-label="email"
-                  required
-                />
-              </li>
-              <li>
-                <label for="message"></label>
-                <textarea
-                  name="message"
-                  id="message"
-                  tabindex="6"
-                  maxlength="500"
-                  rows="5"
-                  placeholder="Write your message here"
-                  aria-label="message"
-                  required
-                ></textarea>
-              </li>
-            </ul>
-            <button class="btn btn-contact" type="submit" tabindex="7">Get it Touch</button>
-          </form>
-        </div>
-      </section>
-      <!-- End of Contact Section -->
-    </div>
-    <!-- End of Main Container -->
-  </body>
-</html>
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.3/font/bootstrap-icons.css"
+    />
+
+    <title>Ghazanfar Portfolio</title>
+  </head>
+
+  <body>
+    <div class="main-container">
+      <div class="top-bar"></div>
+
+      <header class="toolbar">
+        <div class="logo">
+          <a href="#">My Logo</a>
+        </div>
+
+        <nav class="navbar">
+          <img src="./images/Union.png" alt="Menu Icon" class="icon menu" />
+          <i class="bi bi-x"></i>
+          <ul class="navbar-list">
+            <li class="navbar-item">
+              <a href="#" tabindex="1">Portfolio</a>
+            </li>

```

Comment:
- [ ] When the portfolio link in the mobile menu is clicked, it doesn't take me to the correct part of the page. All other links are working except this one. Please modify it 🙏